backend/qemu.pm

```diff
@@ -338,7 +338,7 @@ sub save_memory_dump {
 
     if ($compress_method eq 'xz') {
         if (defined which('xz')) {
-            system(('xz', '-T', $compress_threads, "-v$compress_level", "ulogs/$filename"));
+            runcmd('xz', '--no-warn', '-T', $compress_threads, "-v$compress_level", "ulogs/$filename");
```

Comment:
Interesting, my version of xz, openSUSE Leap 15.1, does not mention this option on `xz --help` but the manpage does.